documentation-site/pages/components/select.mdx

```diff
@@ -0,0 +1,41 @@
+<!--
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+import Example from '../../components/example';
+import API from '../../components/api';
+import Layout from '../../components/layout';
+
+import SelectBasic from 'examples/select/basic.js';
+import SelectSinglePickSearch from 'examples/select/search-single-pick.js';
+import SelectMultiPickSearch from 'examples/select/search-multi-pick.js';
+import SelectWithManyOptions from 'examples/select/with-many-options.js';
+
+export default Layout;
+
+# Select
+
```

Comment:
can we add a description? I have been referencing https://material.io/design/components/menus.html#